src/components/TextField/TextField.scss

```diff
@@ -7,6 +7,10 @@ $stacking-order: (
   backdrop: 10,
 );
 
+.red {
+  color: red;
```

Comment:
is there a specific red we should make this? (Todd question)

Reply:
![screen shot 2017-09-27 at 1 08 22 pm](https://user-images.githubusercontent.com/20175863/30935376-561bec94-a385-11e7-88a0-9c8c598eb26c.png)
todd asked for red, but my guess is that the exact shade would matter too much in this case